src/fu-util.c

```diff
@@ -4869,6 +4903,14 @@ main(int argc, char *argv[])
 	     /* TRANSLATORS: command line option */
 	     N_("Answer yes to all questions"),
 	     NULL},
+	    {"fast-fail",
+	     'y',
+	     0,
+	     G_OPTION_ARG_NONE,
+	     &priv->fast_fail,
+	     /* TRANSLATORS: command line option */
+	     N_("Return status as soon as one update has failed"),
+	     NULL},
```

Comment:
I'd argue we want the inverse.  When do you ever want to let it keep going it if fails?  Debugging only I'd think.

Reply:
Ohh I use it for "how many emulations are broken by this change" but you're completely correct. Should I add a [secret?] command line argument (`--keep-going?`) or just rip out the "keep going" functionality completely?

Reply:
How about a secret option only for unsupported build?